services/nats/README.md

```diff
@@ -0,0 +1,23 @@
+# Nats Service
+
+The nats service is the event broker of the system. It distributes events among all other services. This enables other services to communicate asynchrounos.
+
+## Functionality
+
+Other services can `Publish` events to the nats service. Nats will store these events and distribute them to other services eventually. Similary other services can
+`Consume` events from the nats services by registering to a `ConsumerGroup`. Each `ConsumerGroup` is guaranteed to get each event exactly once. In most cases each service will register its own
+`ConsumerGroup` with multiple instances of the same service registering to the same `ConsumerGroup`
```

Comment:
```suggestion

Services can `Publish` events to the nats service and nats will store respectively distribute these events to other services eventually. Services can `Consume` events from the nats services by registering to a `ConsumerGroup`. Each `ConsumerGroup` is guaranteed to get each event exactly once. In most cases, each service will register it's own `ConsumerGroup` with multiple instances of the same service registering to the same `ConsumerGroup`.
```
...nats will store...
temporary store?
phyically on disk, in memory only...?

...eventually
I am not clear about what that means as this could be interpreted as "if you like or not" 😅

Reply:
1) on disc. See `Persistence` part

2) eventually means `At an unspecified future time`, `In an eventual manner; finally; ultimately.`, `In the end`